<!DOCTYPE html>
<html lang="en-us">

  <head>
  <!-- <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta http-equiv="content-type" content="text/html; charset=utf-8"> -->

  <!-- Enable responsiveness on mobile devices-->
  <!-- <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1"> -->

  <title>
    
       &middot; iobio
    
  </title>

  <!-- CSS -->
  <link rel="stylesheet" href="/public/css/poole.css">
  <link rel="stylesheet" href="/public/css/syntax.css">
  <link rel="stylesheet" href="/public/css/lanyon.css">
  <link rel="stylesheet" href="/public/css/iobio.viz.min.css">
  <link rel="stylesheet" href="/public/js/jquery.min.js">
  <!-- <link rel="stylesheet" href="/public/css/bootstrap/css/bootstrap.min.css">   -->
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700%7CPT+Sans:400">
  <link href='http://fonts.googleapis.com/css?family=Quicksand:300,400' rel='stylesheet' type='text/css'>
  <link href='http://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet' type='text/css'>
  <link href='http://fonts.googleapis.com/css?family=Overlock+SC' rel='stylesheet' type='text/css'>
  <link href='http://fonts.googleapis.com/css?family=Open+Sans:300,400' rel='stylesheet' type='text/css'>
  <link href="/public/js/video-js/video-js.css" rel="stylesheet">
  <link rel="stylesheet" href="/public/css/site.css">

  <!-- JS -->
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="/public/js/video-js/video.js"></script>
  <script src="/public/js/d3.min.js"></script>
  <script src="/public/js/iobio.viz.min.js"></script>

  <!-- Icons -->
  <!-- <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/public/apple-touch-icon-precomposed.png"> -->
  <link rel="icon" type="image/png" href="/public/favicon.ico?v=5">
  <link rel="shortcut icon" href="/public/favicon.ico?v=5">

  <!-- RSS -->
  <link rel="alternate" type="application/rss+xml" title="RSS" href="/atom.xml">

  <!-- tracking code -->
  <script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

    ga('create', 'UA-47481907-1', 'auto');
    ga('send', 'pageview');
  </script>
</head>


  <body>
    <div class="masthead blog-masthead">   
      <div class='masthead-container'>     
        <h3 class="masthead-title">
          <a href="/" title="Home">iobio</a>            
        </h3>
        <div id="menu">
          <ul>
              
              
              
  

  

  

  

  

  

  
    
      <li><a href="/">Home</a></li>
    
  

  
    
      <li><a href="/applications.html">Applications</a></li>
    
  

  
    
      <li><a href="/developers.html">Developers</a></li>
    
  

  
    
      <li><a href="/blog.html">Blog</a></li>
    
  

  
    
      <li><a href="/about.html">About us</a></li>
    
  

  
    
      <li><a href="/support.html">Support Us</a></li>
    
  



          </ul>
        </div>
        <div style="clear:both"></div>
      </div>        
    </div>

    <div class="container blog-container">
      <div style="width:100%; text-align:center; font-size: 60px; font-family: 'Open Sans'; color: rgb(70,70,70); margin-top:20px" >gene.iobio Blog</div>
<div style="width:40px; border-bottom: 1px solid rgb(70,70,70); margin-left:auto;margin-right:auto"></div>
<div class="posts">
  <ul>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2017/09/29/gene_2.5.0/"><img src=/public/images/blog/gene_2.5.0/banner.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Tony DiSera ON Sep 29, 2017</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Tony_DiSera.png"/>
        <div class='snippet-title'><a href="/2017/09/29/gene_2.5.0/">ClinVar variants, gnomAD, affected status</a></div>
        <div class='snippet-subtitle'>What's new in gene.iobio 2.5.0</div>
      </div>

      <!-- <div class="entry">
        <p>Release 2.5.0 introduces some important improvements to the gene.iobio app.  Key among these is the new ‘ClinVar Variants’ panel that shows all known ClinVar variants for a gene.  There are also a number of exciting features in this release including gnomAD allele frequencies and advanced variant filtering based on parent and sibling affected status.</p>


      </div> -->

      <!-- <a href="/2017/09/29/gene_2.5.0/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2017/09/27/loading-data/"><img src=/public/images/blog/loading_data/blog_image_data.jpg></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Tony DiSera ON Sep 27, 2017</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Tony_DiSera.png"/>
        <div class='snippet-title'><a href="/2017/09/27/loading-data/">Loading data into gene.iobio</a></div>
        <div class='snippet-subtitle'>Loading your variant and alignment files</div>
      </div>

      <!-- <div class="entry">
        <p>Gene.iobio accesses variant and sequence alignment files to perform real-time analysis.  This blog post explains how to load your data files.</p>


      </div> -->

      <!-- <a href="/2017/09/27/loading-data/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2017/07/11/coverage/"><img src=/public/images/blog/coverage/main.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Alistair Ward ON Jul 11, 2017</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Alistair_Ward.png"/>
        <div class='snippet-title'><a href="/2017/07/11/coverage/">Coverage analysis</a></div>
        <div class='snippet-subtitle'>Identifying problems with coverage</div>
      </div>

      <!-- <div class="entry">
        <p>A new version of <a href="http://gene.iobio.io/" target="_blank">gene.iobio</a> is up, and this one has a really big feature addition that is really powerful. This update will help identify possible false negative calls due to data insufficiency, and is extremely important in any diagnostic analysis.</p>


      </div> -->

      <!-- <a href="/2017/07/11/coverage/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2017/04/26/gene_2.3.0/"><img src=/public/images/blog/gene_2.3.0/main.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Alistair Ward ON Apr 26, 2017</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Alistair_Ward.png"/>
        <div class='snippet-title'><a href="/2017/04/26/gene_2.3.0/">Gene.iobio 2.3.0</a></div>
        <div class='snippet-subtitle'>Output data from gene.iobio</div>
      </div>

      <!-- <div class="entry">
        <p>A new version of <a href="http://gene.iobio.io/" target="_blank">gene.iobio</a> is now up and running, featuring several improvements that are another big step forwards!</p>


      </div> -->

      <!-- <a href="/2017/04/26/gene_2.3.0/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2017/02/27/gene_2.2.0/"><img src=/public/images/blog/gene_2.2.0/main.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Alistair Ward ON Feb 27, 2017</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Alistair_Ward.png"/>
        <div class='snippet-title'><a href="/2017/02/27/gene_2.2.0/">Gene.iobio 2.2.0</a></div>
        <div class='snippet-subtitle'>Major update to gene.iobio</div>
      </div>

      <!-- <div class="entry">
        <p>We are really happy to announce the latest release of <a href="http://gene.iobio.io/" target="_blank">gene.iobio</a>; version 2.2.0. We have made a major upgrade to the user interface, as well as implementing some new features that really boost the power of analysis with gene.iobio. In this post, I am going to highlight the major changes and recommend that you take a look at the linked tutorials, which demonstrate the power of these new features. The improved filtering functionality significantly enhances gene.iobio, so I strongly urge you read that section! We have also put out a <a href="https://www.youtube.com/watch?v=QiJ7wuN8LYQ" target="_blank">webcast</a> to highlight some of these changes.</p>


      </div> -->

      <!-- <a href="/2017/02/27/gene_2.2.0/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2017/01/23/ref_genomes/"><img src=/public/images/blog/grch38/main.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Alistair Ward ON Jan 23, 2017</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Alistair_Ward.png"/>
        <div class='snippet-title'><a href="/2017/01/23/ref_genomes/">Reference genomes</a></div>
        <div class='snippet-subtitle'>Using alternate references</div>
      </div>

      <!-- <div class="entry">
        <p>The latest release of <a href="http://gene.iobio.io/" target="_blank">gene.iobio</a> includes the ability to use the GRCh38 human reference genome for analysis. This is only a quick post, because we will soon be releasing a major update to gene.iobio, version 2.2.0, which includes significant updates to both the appearance and functionality.</p>


      </div> -->

      <!-- <a href="/2017/01/23/ref_genomes/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2016/11/03/clinvar/"><img src=/public/images/blog/clinvar_viewer/main.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Alistair Ward ON Nov 3, 2016</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Alistair_Ward.png"/>
        <div class='snippet-title'><a href="/2016/11/03/clinvar/">Viewing ClinVar variants</a></div>
        <div class='snippet-subtitle'>What pathogenic variants do we know?</div>
      </div>

      <!-- <div class="entry">
        <p>As part of the analysis of a clinical dataset, we needed to answer a very simple question; is this exon important? In answering this question, we inadvertantly created a ClinVar variant viewer using gene.iobio.</p>


      </div> -->

      <!-- <a href="/2016/11/03/clinvar/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2016/09/22/gene2.1/"><img src=/public/images/blog/gene2.1/main.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Alistair Ward ON Sep 22, 2016</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Alistair_Ward.png"/>
        <div class='snippet-title'><a href="/2016/09/22/gene2.1/">Gene.iobio version 2.1</a></div>
        <div class='snippet-subtitle'>A new version of gene.iobio is now available</div>
      </div>

      <!-- <div class="entry">
        <p>It has been a little while since version 2 of <a href="http://gene.iobio.io" target="_blank">gene.iobio</a> was released, but we have now updated to version 2.1. A lot of work has gone into this update (specifically by Tony Di Sera), including the addition of some powerful features that we're sure you will find useful. In the coming weeks, we will describe these features in more detail, but some of the main updates are summarised here.</p>


      </div> -->

      <!-- <a href="/2016/09/22/gene2.1/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2016/08/03/bookmarks/"><img src=/public/images/blog/bookmarks/main.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Alistair Ward ON Aug 3, 2016</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Alistair_Ward.png"/>
        <div class='snippet-title'><a href="/2016/08/03/bookmarks/">Using bookmarks</a></div>
        <div class='snippet-subtitle'>What are bookmarks and why should I use them?</div>
      </div>

      <!-- <div class="entry">
        <h2 id="general-use">General use</h2>


      </div> -->

      <!-- <a href="/2016/08/03/bookmarks/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2016/05/31/museum-exhibit-gene/"><img src=/public/images/blog/exhibit/exhibit_announcement.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Tony DiSera ON May 31, 2016</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Tony_DiSera.png"/>
        <div class='snippet-title'><a href="/2016/05/31/museum-exhibit-gene/">Gene.iobio museum exhibit</a></div>
        <div class='snippet-subtitle'>Gene.iobio is featured in Genome - Unlocking Life's Code museum exhibition</div>
      </div>

      <!-- <div class="entry">
        <p>Gene.iobio is currently featured at the Natural History Museum of Utah's exhibit <a href="https://nhmu.utah.edu/genome">Genome:  Unlocking Life's Code</a>.   NHGRI partnered with the Smithsonian's National Museum of Natural History (NMNH) to create an <a href="https://unlockinglifescode.org/about">exhibit</a> that chronicles the advancements in genomics and showcases how these discoveries impact people's lives.</p>


      </div> -->

      <!-- <a href="/2016/05/31/museum-exhibit-gene/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2016/03/30/phenolyzer/"><img src=/public/images/blog/phenolyzer/main.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Alistair Ward ON Mar 30, 2016</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Alistair_Ward.png"/>
        <div class='snippet-title'><a href="/2016/03/30/phenolyzer/">Phenolyzer</a></div>
        <div class='snippet-subtitle'>How do I generate phenotype driven gene lists?</div>
      </div>

      <!-- <div class="entry">
        <p>A <a href="http://iobio.io/2016/03/28/multi-gene/" target="_blank">previous post</a> in the <a href="http://gene.iobio.io" target="_blank">gene.iobio</a> version 2 blog series focused on analyzing multiple genes simultaneously. This functionality has really increased the power of <a href="http://gene.iobio.io" target="_blank">gene.iobio</a>, making it a lot easier to analyze a set of genes and identify candidate disease variants. The problem is, though, that we still need to generate a list of genes to analyze. As described in the <a href="http://iobio.io/2016/03/29/acmg56/" target="_blank">previous post</a>, we can start with the list of 56 genes identified by the ACMG, but what would be ideal, would be start with a phenotype and generate a gene list from there. Fortunately, that is exactly what we can do!</p>


      </div> -->

      <!-- <a href="/2016/03/30/phenolyzer/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2016/03/29/acmg56/"><img src=/public/images/blog/acmg56/main.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Alistair Ward ON Mar 29, 2016</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Alistair_Ward.png"/>
        <div class='snippet-title'><a href="/2016/03/29/acmg56/">ACMG 56 genes</a></div>
        <div class='snippet-subtitle'>Analyzing genes identified by the ACMG</div>
      </div>

      <!-- <div class="entry">
        <p>In March 2013, an <a href="https://www.acmg.net/" target="_blank">The American College of Medical Genetics and Genomics (ACMG)</a> working group released a list of 57 genes (later revised to 56), that harbour mutations that when identified, could lead to severe outcomes, but were medically actionable. These genes are implicated in a number of disorders, including inherited cancers, inherited cardiac diseases, connective tissue diseases affecting the cardiovascular system, familial hypercholesterolemia, and malignant hyperthermia susceptibility. The ACMG also generated a list of <a href="http://www.ncbi.nlm.nih.gov/pubmed/23788249" target="_blank">recommendations</a> that state that known pathogenic (or in some cases expected pathogenic) variants discovered in any of these genes should be reported regardless of an individuals age. These recommendations referred to what are now known as secondary findings in an individual undergoing exome or genome sequencing.</p>


      </div> -->

      <!-- <a href="/2016/03/29/acmg56/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2016/03/28/multi-gene/"><img src=/public/images/blog/multiple-genes/main.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Alistair Ward ON Mar 28, 2016</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Alistair_Ward.png"/>
        <div class='snippet-title'><a href="/2016/03/28/multi-gene/">Multi gene analysis</a></div>
        <div class='snippet-subtitle'>How do I analyze lists of genes, rather than a single gene?</div>
      </div>

      <!-- <div class="entry">
        <p>The <a href="http://gene.iobio.io" target="_blank">gene.iobio</a> version 2 blog series marches on with a discussion of multi-gene analysis. We think this is a major advancement and takes <a href="http://gene.iobio.io" target="_blank">gene.iobio</a> from a cool way of visualizing integrated data in a single gene, to a powerful method of performing analysis across sets of relevant genes. A following post in the series will look at generating gene lists based on phenotype using <a href="http://phenolyzer.usc.edu/" target="_blank">Phenolyzer</a>, which really helps direct analysis.</p>


      </div> -->

      <!-- <a href="/2016/03/28/multi-gene/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2016/03/21/gene-sets/"><img src=/public/images/blog/gene-sets/main.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Alistair Ward ON Mar 21, 2016</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Alistair_Ward.png"/>
        <div class='snippet-title'><a href="/2016/03/21/gene-sets/">GENCODE or RefSeq?</a></div>
        <div class='snippet-subtitle'>Which gene set and transcript should I use?</div>
      </div>

      <!-- <div class="entry">
        <p>Continuing the blog series accompanying the <a href="http://gene.iobio.io" target="_blank">gene.iobio</a> version 2 release, we will discuss how to choose between the <a href="http://www.gencodegenes.org" target="_blank">GENCODE</a> and <a href="http://www.ncbi.nlm.nih.gov/RefSeq" target="_blank">RefSeq</a> transcript sets and then different gene transcripts within each set. There are a lot of interesting discussions on the effects of using one or the other gene set. For example, this <a href="http://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-16-S8-S2" target="_blank">paper</a> has a more detailed discussion on the two sets, and this <a href="http://genomemedicine.biomedcentral.com/articles/10.1186/gm543" target="_blank">paper</a> discusses the differences between functional annotations achieved using different software and transcripts.</p>


      </div> -->

      <!-- <a href="/2016/03/21/gene-sets/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2016/03/16/filter-status/"><img src=/public/images/blog/filter_status/main.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Alistair Ward ON Mar 16, 2016</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Alistair_Ward.png"/>
        <div class='snippet-title'><a href="/2016/03/16/filter-status/">What is the filter status in gene.iobio?</a></div>
        <div class='snippet-subtitle'>Only display desired variants</div>
      </div>

      <!-- <div class="entry">
        <p>This is the second in a <a href="http://iobio.io/2016/03/14/gene-version2/index.html#v2posts" target="_blank">series of blog posts</a> describing updates in the version 2.0 release of <a href="http://gene.iobio.io" target="_blank">gene.iobio</a>. In this post, we discuss the concept of <strong><i>filter status</i></strong> and how to display the variants you want to see. If we open the example <a href="http://gene.iobio.io/?rel0=proband&amp;rel1=mother&amp;rel2=father&amp;name0=&amp;vcf0=https://s3.amazonaws.com/iobio/gene/wgs_platinum/NA12878.platinum.vcf.gz&amp;affectedSibs=&amp;unaffectedSibs=&amp;genes=&amp;gene=BRCA1" target="_blank">data set</a>, click on the 'Filter' button to the left, you will see the 'Filter status' at the top of the left panel.</p>


      </div> -->

      <!-- <a href="/2016/03/16/filter-status/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2016/03/14/gene-version2/"><img src=/public/images/blog/gene_version2/main.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Alistair Ward ON Mar 14, 2016</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Alistair_Ward.png"/>
        <div class='snippet-title'><a href="/2016/03/14/gene-version2/">Gene.iobio version 2.0</a></div>
        <div class='snippet-subtitle'>Release of version 2.0 of gene.iobio with major new functionality</div>
      </div>

      <!-- <div class="entry">
        <p>We are thrilled to announce the release of <a href="http://gene.iobio.io">gene.iobio</a> version 2.0! This new release features a major facelift and lots of new functionality that significantly enhances the power and usability of <a href="http://gene.iobio.io">gene.iobio</a>. The single most important enhancement is that we have now moved from being a single gene to a multi-gene assessment tool, with integrated methods of generating and importing gene lists, including manual import and phenotype driven list generation with <a href="http://phenolyzer.usc.edu">Phenolyzer</a>. Below, we can see where genes being investigated can be selected and the transcript set can be switched between Gencode and RefSeq.</p>


      </div> -->

      <!-- <a href="/2016/03/14/gene-version2/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2015/09/29/gene-iobio-exome-seq/"><img src=/public/images/blog/gene_iobio_exome_featured_image.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Alistair Ward ON Sep 29, 2015</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Alistair_Ward.png"/>
        <div class='snippet-title'><a href="/2015/09/29/gene-iobio-exome-seq/">Latest Exome sequencing in gene.iobio</a></div>
        <div class='snippet-subtitle'>Analyzing a trio with iobio</div>
      </div>

      <!-- <div class="entry">
        <p>Exome data has unique characteristics, which affects the way the data is checked and analyzed. In this post, we will use exome data from a 1000 Genomes project trio to highlight some iobio features that are particularly useful when analyzing exome data.</p>

<p>The first step in the analysis is to ensure that the data quality is sufficiently high. We use bam.iobio to check the global statistics of the alignment files, which we can view <a href="http://bam.iobio.io/?bam=https://s3.amazonaws.com/iobio/1000gSV/NA19238.mapped.ILLUMINA.bwa.YRI.exome.20130415.bam&amp;region=1" target="_new">here</a>.</p>

<div style="text-align:center">
	<img src="/public/images/blog/bam_iobio_exome_data.png" style="width:80%" />
</div>

<p>A noticeable problem with this view for exome alignments is that since the majority of the genome has zero coverage, the read coverage chart is dominated by the zero coverage bases. To focus on the exome only, we can select the ‘Default Bed’ option (highlighted in the above plot) to use the 1000 Genomes human exome targets file to focus sampling in these regions.</p>

<div style="text-align:center">
	<img src="/public/images/blog/bam_iobio_exome_data_coverage_w_bed.png" style="width:70%" />
</div>

<p>When only considering the targeted exome, the coverage is encouragingly generally greater than 60X.</p>

<p>Now, we want to look at the alignments and the variant calls for the trio. We will look at a few different genes, beginning with <a href="http://gene.iobio.io/?rel0=proband&amp;rel1=mother&amp;rel2=father&amp;gene=BRCA2&amp;name0=NA19240&amp;bam0=https://s3.amazonaws.com/iobio/1000gSV/NA19240.mapped.ILLUMINA.bwa.YRI.exome.20130415.bam&amp;name1=NA19238&amp;bam1=https://s3.amazonaws.com/iobio/1000gSV/NA19238.mapped.ILLUMINA.bwa.YRI.exome.20130415.bam&amp;name2=NA19239&amp;bam2=https://s3.amazonaws.com/iobio/1000gSV/NA19239.mapped.ILLUMINA.bwa.YRI.exome.20130415.bam&amp;vcf0=https://s3.amazonaws.com/iobio/1000gSV/exome-trio.vcf.gz&amp;sample0=NA19240&amp;vcf1=https://s3.amazonaws.com/iobio/1000gSV/exome-trio.vcf.gz&amp;sample1=NA19238&amp;vcf2=https://s3.amazonaws.com/iobio/1000gSV/exome-trio.vcf.gz&amp;sample2=NA19239" target="_new">BRCA2</a>. We know that the 1000 Genomes samples were over 18 years of age and healthy at the time of DNA collection, but we can see that the father is carrying a heterozygous variant identified as possibly damaging by PolyPhen and deleterious by Sift in one of the exons. Snpeff identifies this variant as having a moderate impact and we can also see that the child (identified as the proband in this trio) has inherited this allele from her father. This variant appears as the highest ranked in the ‘Rank Variants’ table.</p>

<div style="text-align:center">
	<img src="/public/images/blog/gene_iobio_exome_data.png" style="width:100%" />
</div>

<p>Now, let’s take a look at the <a href="http://gene.iobio.io/?rel0=proband&amp;rel1=mother&amp;rel2=father&amp;gene=CFTR&amp;name0=NA19240&amp;bam0=https://s3.amazonaws.com/iobio/1000gSV/NA19240.mapped.ILLUMINA.bwa.YRI.exome.20130415.bam&amp;name1=NA19238&amp;bam1=https://s3.amazonaws.com/iobio/1000gSV/NA19238.mapped.ILLUMINA.bwa.YRI.exome.20130415.bam&amp;name2=NA19239&amp;bam2=https://s3.amazonaws.com/iobio/1000gSV/NA19239.mapped.ILLUMINA.bwa.YRI.exome.20130415.bam&amp;vcf0=https://s3.amazonaws.com/iobio/1000gSV/exome-trio.vcf.gz&amp;sample0=NA19240&amp;vcf1=https://s3.amazonaws.com/iobio/1000gSV/exome-trio.vcf.gz&amp;sample1=NA19238&amp;vcf2=https://s3.amazonaws.com/iobio/1000gSV/exome-trio.vcf.gz&amp;sample2=NA19239" target="_new">CFTR gene</a>.</p>

<div style="text-align:center">
	<img src="/public/images/blog/gene_iobio_exome_data_cftr.png" style="width:100%" />
</div>

<p>When we open gene.iobio in the CFTR gene, we are presented with the canonical transcript. We can immediately see that there is coverage in all three samples in what appears to be an intronic location. This situation is likely the result of an exon present in an alternative transcript for this gene. To check, we can use the ‘Transcript’ drop-down menu and look at all the other available transcripts. The processed transcript (i.e. a transcript that does not contain an open reading frame) highlighted in the figure below appears to have an exon in the location with coverage.</p>

<div style="text-align:center">
	<img src="/public/images/blog/gene_iobio_transcript_menu_cftr.png" style="width:60%" />
</div>

<p>When selected, we see that this coverage peak indeed fell within this exon.</p>

<div style="text-align:center">
	<img src="/public/images/blog/gene_iobio_alt_transcript_cftr.png" style="width:40%" />
</div>

<p>Finally, we will look at the <a href="http://gene.iobio.io/?rel0=proband&amp;rel1=mother&amp;rel2=father&amp;gene=ABCB10&amp;name0=NA19240&amp;bam0=https://s3.amazonaws.com/iobio/1000gSV/NA19240.mapped.ILLUMINA.bwa.YRI.exome.20130415.bam&amp;name1=NA19238&amp;bam1=https://s3.amazonaws.com/iobio/1000gSV/NA19238.mapped.ILLUMINA.bwa.YRI.exome.20130415.bam&amp;name2=NA19239&amp;bam2=https://s3.amazonaws.com/iobio/1000gSV/NA19239.mapped.ILLUMINA.bwa.YRI.exome.20130415.bam&amp;vcf0=https://s3.amazonaws.com/iobio/1000gSV/exome-trio.vcf.gz&amp;sample0=NA19240&amp;vcf1=https://s3.amazonaws.com/iobio/1000gSV/exome-trio.vcf.gz&amp;sample1=NA19238&amp;vcf2=https://s3.amazonaws.com/iobio/1000gSV/exome-trio.vcf.gz&amp;sample2=NA19239" target="_new">ABCB10 gene</a>.  This gene shows the opposite situation to what we observed in the CFTR gene. The final coding sequence (CDS) in the canonical transcript has no coverage in any of the samples. A quick check of the 1000 Genomes project exome targets bed file reveals that this exon is included and so coverage would be expected at this location. If it were the case that any of the samples harbored a deleterious mutation in this CDS, it would not be identified from this sequencing experiment, since no DNA sequence is present within it. This is a known cause of false negative variants, i.e. variants that are not called when they do exist and can quickly and easily be identified within gene.iobio.</p>

<div style="text-align:center">
	<img src="/public/images/blog/gene_iobio_exome_data_abcb10.png" style="width:70%" />
</div>

      </div> -->

      <!-- <a href="/2015/09/29/gene-iobio-exome-seq/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2015/09/27/multisample-vcf-gene-iobio/"><img src=/public/images/blog/feature_image_allele_counts.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Tony DiSera ON Sep 27, 2015</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Tony_DiSera.png"/>
        <div class='snippet-title'><a href="/2015/09/27/multisample-vcf-gene-iobio/">Latest Gene.iobio features</a></div>
        <div class='snippet-subtitle'>Multi-sample vcf, https support, and allele counts</div>
      </div>

      <!-- <div class="entry">
        <p>The week, we have pushed out some new features in <a href="http://gene.iobio.io" target="_new">gene.iobio</a>.</p>

<h3 id="multi-sample-vcf-files">Multi-sample VCF files</h3>

<p>We now support multi-sample VCF files.  After entering the VCF URL or selecting the VCF file, a dropdown will appear if the file contains multiple samples.</p>

<p>Here we show a VCF file containing 100+ samples.  The sample list can be narrowed down using the type-ahead feature.</p>

<div style="text-align:center">
	<img src="/public/images/blog/multisample_vcf.png" style="width:40%" />
</div>

<p>Here is a screen capture showing a trio loaded from one file.  Even though the file might be quite large, gene.iobio only examines the samples of interest.</p>

<div style="text-align:center">
	<img src="/public/images/blog/multisample_vcf_trio.png" style="width:100%" />
</div>

<h3 id="secure-file-serving">Secure File Serving</h3>

<p>Gene.iobio can now handle files served using the https protocol, ensuring secure communication over the network – a requirement when analyzing sensitive data.</p>

<h3 id="allele-counts">Allele Counts</h3>

<p>Hover over a variant and the tooltip will include a bar for allele counts.  In this example, you can see the allele counts for a trio.  Notice how the parents are heterogous alternates and the proband is homozygous alternate.  The allele counts verify that there is adequate sequence coverage to make these calls.</p>

<div style="text-align:center">
	<img src="/public/images/blog/allele_counts.png" style="width:50%" />
</div>

<p>Stay tuned.  More features are coming soon to gene.iobio including multi-gene navigation and extended trio functionality to analyze variants from unaffected siblings.</p>


      </div> -->

      <!-- <a href="/2015/09/27/multisample-vcf-gene-iobio/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2015/09/07/vep-in-gene-iobio/"><img src=/public/images/blog/veplogo.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Tony DiSera ON Sep 7, 2015</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Tony_DiSera.png"/>
        <div class='snippet-title'><a href="/2015/09/07/vep-in-gene-iobio/">Gene.iobio integrates VEP</a></div>
        <div class='snippet-subtitle'>Polyphen & SIFT classifications, Regulatory annotations, and HGVS & dbSNP ids.</div>
      </div>

      <!-- <div class="entry">
        <p>When we released <a href="http://gene.iobio.io" target="_new">gene.iobio</a> a couple weeks ago, we had a few requests to add Ensembl’s Variant Effect Predictor (VEP) tool.</p>

<div class="tweet-feed" style="padding-bottom:10px;">
	<blockquote class="twitter-tweet" lang="en"><p lang="en" dir="ltr">It will in a new release soon! <a href="https://twitter.com/jxchong">@jxchong</a> <a href="https://twitter.com/brent_p">@brent_p</a></p>&mdash; Gabor Marth (@MarthGabor) <a href="https://twitter.com/MarthGabor/status/629013788931584000">August 5, 2015</a></blockquote>
	<script async="" src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
</div>

<p>And as any good PI does, Gabor was quick to sign us up to do it! After taking his Twitter account away, we made good on the promise and officially added VEP support, which gives us (along with SnpEff) two powerful variant effect predictor tools.</p>

<h3 id="vep-consequence">VEP Consequence</h3>

<p>VEP determines the variant consequence on protein coding, a classification that is analogous to SnpEff’s Effect. In fact, these classifications are highly concordant when evaluating coding variants, but keep in mind that these classifications do diverge in the categories of splicing, frameshift, and stop gain. <a href="http://blog.goldenhelix.com/ajesaitis/the-sate-of-variant-annotation-a-comparison-of-annovar-snpeff-and-vep/">Andrew Jesaitis has written a terrific blog post</a> that explains these differences in his in depth comparison of SnpEff, VEP and Annovar.</p>

<p>We only color variants with either SnpEff or VEP classifications at any given time. You can easily alternate between SnpEff and VEP classifications in the filter panel using the Impact dropdown.</p>

<div style="text-align:center">
	<img src="/public/images/blog/vepblog2.png" style="width:20%" />
</div>

<p>However, both classifications are visible on the variant tooltip displays.</p>

<div style="text-align:center">
	<img src="/public/images/blog/vepblog3.png" style="width:50%" />
</div>

<p>By default, variants are colored based on SnpEff Impact, a high level prioritization scheme on SnpEff Effect, summarizing the categories into four possible levels:  High, Moderate, Modifier and Low.</p>

<div style="text-align:center">
	<img src="/public/images/blog/vepblog4.png" style="width:15%;vertical-align:top" />	
	<img src="/public/images/blog/vepblog5.png" style="width:80%" />
</div>

<p>To color based on SnpEff Effect, click on the Effect link.</p>

<p><img src="/public/images/blog/vepblog6.png" style="width:15%;vertical-align:top" />
<img src="/public/images/blog/vepblog7.png" style="width:80%" /></p>

<p>Now the variants have switched from the SnpEff Impact color scheme (e.g. High, Moderate, Modifier, Low) to the SnpEff Effect color scheme (e.g. missense, frameshift, etc.).  The same color scheme switching can be performed on VEP impact and consequence.</p>

<h3 id="predicting-impact-on-protein-function">Predicting Impact on Protein Function</h3>

<p>Given that non-synonymous SNPs comprise over half of the mutations known to be involved in human inherited diseases, algorithm-based approaches to predicting disruption to protein structure and function are critical to understanding human disease and phenotype <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3572689/pdf/TSWJ2013-675851.pdf">(Wu, Yang 2012)</a>. VEP reports classifications from two such algorithms – <a href="http://sift.jcvi.org/">SIFT</a> and <a href="http://genetics.bwh.harvard.edu/pph2/">PolyPhen</a>.  Both SIFT and PolyPhen analyze sequence homology, whereby substitutions at highly conserved regions are predicted to have higher impact on protein function.  In addition, PolyPhen utilizes sequence annotation to inventory features mapped to the location of the substitution.  PolyPhen also maps to the 3D protein structure to predict if the amino acid substitution will disrupt the hydrophobic core, alter electrostatic interactions, or interfere with ligand actions.</p>

<p>Gene.iobio provides filters based on SIFT and PolyPhen classifications:</p>

<div style="text-align:center">
	<img src="/public/images/blog/vepblog8.png" style="width:20%;vertical-align:top" />
	<img src="/public/images/blog/vepblog9.png" style="width:25%;vertical-align:top" />
</div>

<p>In addition, PolyPhen and SIFT classifications are used to rank the variants.</p>

<div style="text-align:center">
	<img src="/public/images/blog/vepblog10.png" style="width:50%" />
</div>

<p>In this example, the first variant in the ranked list has a PolyPhen classification of ‘Probably Damaging’ and a SIFT classification of ‘Deleterious’, providing additional insight into a moderate impact variant catalogued with conflicting interpretations in ClinVar.</p>

<p>When evaluating a variant’s pathogenicity, it is important to consider the variation type and seek other evidence-based sources in addition to the SIFT and PolyPhen predictions.  In the paper by <a href="http://www.researchgate.net/publication/45273588_Using_SIFT_and_PolyPhen_to_Predict_Loss-of-Function_and_Gain-of-Function_Mutations">Flanagan, et al 2010</a> which tested the predictive value of SIFT and PolyPhen, it was found that the sensitivity was reasonably high, but that the specificity was quite low, especially for gain-of-function mutations. When considering just SNPs or loss-of-function mutations, these tools performed much better.</p>

<h3 id="regulatory-annotations">Regulatory Annotations</h3>

<p>In addition to annotating protein coding variants, VEP also reports on variants that are located in predicted regulatory sites based on the <a href="http://www.ensembl.org/info/genome/funcgen/regulatory_build.html">Ensembl Regulatory Build</a>.  Gene.iobio now allows variant filtering on these regulatory classifications:</p>

<div style="text-align:center">
	<img src="/public/images/blog/vepblog11.png" style="width:20%" />
</div>

<p>And the variant tooltip provides link outs to Ensembl, showing multiple tracks for different cell types.  For example, SNP located in a predicted TF binding site is only active in the Human Embryonic Stem Cell type.</p>

<div style="text-align:center">
	<img src="/public/images/blog/vepblog12.png" style="width:30%;vertical-align:top" />
	<img src="/public/images/blog/vepblog13.png" style="width:65%" />
</div>

<p>For TF motif features, the link out to Ensembl shows the sequence logo along with a frequency matrix from the JASPER core database <a href="http://www.ncbi.nlm.nih.gov/pubmed?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=14681366&amp;ordinalpos=3&amp;itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum">(Mathelier, et al 2013)</a>.</p>

<div style="text-align:center">
	<img src="/public/images/blog/vepblog14.png" style="width:30%;vertical-align:top" />
	<img src="/public/images/blog/vepblog15.png" style="width:65%" />
</div>

<h3 id="variant-identifiers-and-nomenclature">Variant Identifiers and Nomenclature</h3>

<p>VEP also provides the dbSNP identifier (i.e. rs number) as well as the HGVS notation for the variant. The HGVS notation is based on the GENCODE transcript set. Additionally, we plan to incorporate the RefSeq transcript set in the near future.  In gene.iobio, the tooltip for the variant will show both HGVS notation and the dbSNP id, with the latter linking out to the dbSNP webpage for the particular variant.</p>

<div style="text-align:center">
	<img src="/public/images/blog/vepblog16.png" style="width:40%" />
</div>

<h3 id="coming-soon">Coming Soon</h3>

<p>With VEP integration complete, stay tuned for enhancements to easily navigate through a set of genes, support for multi-sample vcfs, color exons based on sequence coverage, and much more.  (VEP is no longer planned!)</p>


      </div> -->

      <!-- <a href="/2015/09/07/vep-in-gene-iobio/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2015/09/03/install-run-tabix/"><img src=/public/images/blog/tabix.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Yi Qiao ON Sep 3, 2015</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Yi_Qiao.png"/>
        <div class='snippet-title'><a href="/2015/09/03/install-run-tabix/">Creating a compressed and indexed VCF</a></div>
        <div class='snippet-subtitle'>The .vcf.gz and .vcf.gz.tbi file pair</div>
      </div>

      <!-- <div class="entry">
        <p>Compressing VCF files with BGZip and indexing it with Tabix is the standard way VCF files are stored, and is the format that iobio apps support. The software to compress and index VCF files is called Tabix, developed by Heng Li at the Broad Institute.</p>

<h3 id="for-macos-systems">For MacOS systems</h3>

<p>We provide a simple Mac App for compressing and indexing vcf files using Tabix.</p>

<ul>
  <li>Download the <a href="http://vcf.iobio.io/tabix/iTabixit.zip">iTabixIt App</a>.</li>
  <li>Double-click to decompress it.</li>
  <li>Download the <a href="http://vcf.iobio.io/tabix/TabixBinary.zip">Tabix and Bgzip binaries</a>.</li>
  <li>Double-click to decompress it in the same parent folder as the iTabixApp.</li>
  <li>Now you should see an icon for the iTabixApp and a folder for the binaries.</li>
</ul>

<div style="text-align:center">
	<img src="/public/images/blog/tabix1.png" style="width:20%" />
</div>

<ul>
  <li>Now double-click on the iTabixIt icon to launch the app.</li>
  <li>Select a .vcf file.  The app will start processing to compress and index the .vcf file.</li>
</ul>

<div style="text-align:center">
	<img src="/public/images/blog/tabix2.png" style="width:50%" />
</div>

<ul>
  <li>When the app is down compressing and indexing the VCF file, green dots will appear next to the completed items.</li>
</ul>

<div style="text-align:center">
	<img src="/public/images/blog/tabix3.png" style="width:50%" />
</div>

<ul>
  <li>Now the .vcf.gz and .vcf.gz.tbi files will be created in the same directory as the .vcf file. Now you are ready to run vcf.iobio or gene.iobio, selecting these two files when prompted.</li>
</ul>

<h3 id="for-linux-based-system">For Linux-based system</h3>

<ul>
  <li>Download the latest tabix jar at <a href="http://sourceforge.net/projects/samtools/files/tabix/">http://sourceforge.net/projects/samtools/files/tabix/</a>.</li>
  <li>Install C compiler and the ‘make’ utility with the package management system of your specific Linux distribution. In the case of Ubuntu, the following command should suffice:</li>
</ul>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>	$ sudo apt-get install build-essential
	$ tar -xjf tabix-0.2.6.tar.bz2
	$ cd tabix-0.2.6
	$ make
</code></pre></div></div>

<ul>
  <li>Once you have successfully compiled Tabix (and bgzip), you can compress your vcf files with the following command:</li>
</ul>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>	$ /path/to/tabix/bgzip myvcf.vcf
	$ /path/to/tabix/tabix -p vcf myvcf.vcf.gz
</code></pre></div></div>

<ul>
  <li>These two commands will create two new files: the compressed vcf (.vcf.gz) and the index (.vcf.gz.tbi). Now you are ready to run vcf.iobio or gene.iobio, selecting these two files when prompted.</li>
</ul>

<p>The manual page for Tabix and BGZip can be found at <a href="http://samtools.sourceforge.net/tabix.shtml">http://samtools.sourceforge.net/tabix.shtml</a>.</p>


      </div> -->

      <!-- <a href="/2015/09/03/install-run-tabix/" class="read-more">Read More</a> -->
    </li>
  
    <li class="snippet">

      <div class='snippet-thumbnail'><a href="/2015/08/04/gene.iobio_released/"><img src=/public/images/blog/gene.iobio.png></img></a></div>
      <div class='snippet-text'>
        <div class='snippet-byline'>BY Chase Miller ON Aug 4, 2015</div>
        <div></div>
        <img class='snippet-headshot' src="/public/images/team/Chase_Miller.png"/>
        <div class='snippet-title'><a href="/2015/08/04/gene.iobio_released/">New app gene.iobio</a></div>
        <div class='snippet-subtitle'>interrogate potential disease-causing variants</div>
      </div>

      <!-- <div class="entry">
        <p>We’ve been working on a new real-time interactive iobio analysis tool for the last few months and today are excited to officially release <a href="http://gene.iobio.io">gene.iobio</a>. <a href="http://gene.iobio.io">Gene.iobio</a> is designed to help medical and clinical researchers hunt for disease-causing genetic variants through a combination of real-time genomic data analysis and intuitive visualization. The quickest way to get an idea of the features and functionality is to watch the <a href="https://www.youtube.com/watch?v=Pj5WinfmzAs">intro screencast</a>, quite professionally voiced over by our own Alistair Ward!</p>


      </div> -->

      <!-- <a href="/2015/08/04/gene.iobio_released/" class="read-more">Read More</a> -->
    </li>
  
  </ul>
</div>
    </div>

    <div class="footer">
  
    <div class="logos">
      <div style="position:absolute; left: 100px; top:0px; font-size:50px;font-family:'Overlock SC', cursive;"><a href="http://marthlab.org" style="color:black; text-decoration:none;">Marthlab</a></div>
      <a href="http://ucgd.genetics.utah.edu/"><img src="/public/images/ustar-ucgd-logo.jpg" style="height:60px;"/></a>
      <a href="http://www.genetics.utah.edu/"><img src="/public/images/genetics_mainlogo3_lrg.png" style="height:50px;position:absolute;right:0px;top:7px"/></a>
    </div>
  
  <div class="developers">
        <div class='title'>Developed in the <a href="http://marthlab.org/">Marth Lab by</a></div>

        <div class="members">
          <span style="margin-right:15px">
            <a href="https://github.com/chmille4"><img src="/public/images/github_2d8fc1.png"/></a>
            Chase Miller
          </span>
          <span style="margin-right: 15px">
            <a href="https://github.com/yiq"><img src="/public/images/github_2d8fc1.png"/></a>
            Yi Qiao
          </span>
          <span style="margin-right: 15px">
            <a href="https://github.com/tonydisera"><img src="/public/images/github_2d8fc1.png"/></a>
            Tony DiSera
          </span>
          <span style="margin-right: 15px">
            <a href="https://github.com/alistairnward"><img src="/public/images/github_2d8fc1.png"/></a>
            Alistair Ward
          </span>
        </div>
    </div>
</div>
  </body>
</html>
