<!DOCTYPE html>
<html lang="en-us">

  <head>
  <!-- <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta http-equiv="content-type" content="text/html; charset=utf-8"> -->

  <!-- Enable responsiveness on mobile devices-->
  <!-- <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1"> -->

  <title>
    
      Phenolyzer &middot; iobio
    
  </title>

  <!-- CSS -->
  <link rel="stylesheet" href="/public/css/poole.css">
  <link rel="stylesheet" href="/public/css/syntax.css">
  <link rel="stylesheet" href="/public/css/lanyon.css">
  <link rel="stylesheet" href="/public/css/iobio.viz.min.css">
  <link rel="stylesheet" href="/public/js/jquery.min.js">
  <!-- <link rel="stylesheet" href="/public/css/bootstrap/css/bootstrap.min.css">   -->
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700%7CPT+Sans:400">
  <link href='http://fonts.googleapis.com/css?family=Quicksand:300,400' rel='stylesheet' type='text/css'>
  <link href='http://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet' type='text/css'>
  <link href='http://fonts.googleapis.com/css?family=Overlock+SC' rel='stylesheet' type='text/css'>
  <link href='http://fonts.googleapis.com/css?family=Open+Sans:300,400' rel='stylesheet' type='text/css'>
  <link href="/public/js/video-js/video-js.css" rel="stylesheet">
  <link rel="stylesheet" href="/public/css/site.css">

  <!-- JS -->
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="/public/js/video-js/video.js"></script>
  <script src="/public/js/d3.min.js"></script>
  <script src="/public/js/iobio.viz.min.js"></script>

  <!-- Icons -->
  <!-- <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/public/apple-touch-icon-precomposed.png"> -->
  <link rel="icon" type="image/png" href="/public/favicon.ico?v=5">
  <link rel="shortcut icon" href="/public/favicon.ico?v=5">

  <!-- RSS -->
  <link rel="alternate" type="application/rss+xml" title="RSS" href="/atom.xml">

  <!-- tracking code -->
  <script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

    ga('create', 'UA-47481907-1', 'auto');
    ga('send', 'pageview');
  </script>
</head>


  <body>
    <div class="masthead blog-masthead">   
      <div class='masthead-container'>     
        <h3 class="masthead-title">
          <a href="/" title="Home">iobio</a>            
        </h3>
        <div id="menu">
          <ul>
              
              
              
  

  

  

  

  

  

  
    
      <li><a href="/">What is it?</a></li>
    
  

  
    
      <li><a href="/applications.html">Applications</a></li>
    
  

  
    
      <li><a href="/developers.html">Developers</a></li>
    
  

  
    
      <li><a href="/blog.html">Blog</a></li>
    
  

  
    
      <li><a href="/about.html">About us</a></li>
    
  

  
    
      <li><a href="/support.html">Support Us</a></li>
    
  



          </ul>
        </div>
        <div style="clear:both"></div>
      </div>        
    </div>

    <div class="container blog-container">
      <div class="post post-main" style="margin-top: 100px;">
  <div class="post-title">Phenolyzer</div>
  <div style="line-height: 17px; margin-top: -9px">
    <div class="post-date">30 Mar 2016</div>
    <div class="post-author"><a href="http://marthlab.org/people.html">Alistair Ward</a></div>
    <div class='post-headshot'>
      <a href="http://marthlab.org/people.html">
      <img src="/public/images/team/Alistair_Ward.png"/></a>
    </div>
  </div>
  <div class='post-image'> <img src=/public/images/blog/phenolyzer/main.png></img></div>
  <div class='post-content'>
    <p>A <a href="http://iobio.io/2016/03/28/multi-gene/" target="_blank">previous post</a> in the <a href="http://gene.iobio.io" target="_blank">gene.iobio</a> version 2 blog series focused on analyzing multiple genes simultaneously. This functionality has really increased the power of <a href="http://gene.iobio.io" target="_blank">gene.iobio</a>, making it a lot easier to analyze a set of genes and identify candidate disease variants. The problem is, though, that we still need to generate a list of genes to analyze. As described in the <a href="http://iobio.io/2016/03/29/acmg56/" target="_blank">previous post</a>, we can start with the list of 56 genes identified by the ACMG, but what would be ideal, would be start with a phenotype and generate a gene list from there. Fortunately, that is exactly what we can do!</p>

<p>In this post, we will use the modified <a href="http://gene.iobio.io/?rel0=proband&amp;rel1=mother&amp;rel2=father&amp;name0=&amp;vcf0=https://s3.amazonaws.com/iobio/gene/wgs_platinum/platinum-trio.vcf.gz&amp;bam0=https://s3.amazonaws.com/iobio/gene/wgs_platinum/NA12878.bam&amp;name1=&amp;vcf1=https://s3.amazonaws.com/iobio/gene/wgs_platinum/platinum-trio.vcf.gz&amp;bam1=https://s3.amazonaws.com/iobio/gene/wgs_platinum/NA12892.bam&amp;name2=&amp;vcf2=https://s3.amazonaws.com/iobio/gene/wgs_platinum/platinum-trio.vcf.gz&amp;bam2=https://s3.amazonaws.com/iobio/gene/wgs_platinum/NA12891.bam&amp;sample0=NA12878&amp;sample1=NA12891&amp;sample2=NA12892&amp;genes=RAI1&amp;gene=RAI1" target="_blank">Platinum trio</a> dataset and assume this data originates from a family trio where the child is suffering from <i>lactic acidosis</i>. If we know a lot about this phenotype, we may know of some likely genes where we can start our analysis, but what if we do not? Or what if these genes turn up no candidate variants? Using <a href="http://phenolyzer.usc.edu/" target="_blank">Phenolyzer</a>, a tool developed in the lab of <a href="http://genomics.usc.edu/" target="_blank">Kai Wang</a> at <a href="http://www.usc.edu/" target="_blank">USC</a>, we can use the phenotype to generate a gene list. For more information on this tool, or other software developed in this lab, go <a href="http://genomics.usc.edu/research" target="_blank">here</a>.</p>

<p>To use <a href="http://phenolyzer.usc.edu/" target="_blank">Phenolyzer</a>, click on the 'Genes' button to the left of the <a href="http://gene.iobio.io" target="_blank">gene.iobio</a> screen and then enter your phenotype terms where indicated.</p>

<div><img src="/public/images/blog/phenolyzer/phenolyzer.png" style="width:55%; margin:20px 120px;" /></div>

<p>In this case, we enter <i>lactic acidosis</i>, click 'Run' and wait for <a href="http://phenolyzer.usc.edu/" target="_blank">Phenolyzer</a> to do its magic! Very briefly, <a href="http://phenolyzer.usc.edu/" target="_blank">Phenolyzer</a> works by trying to match the entered phenotype terms to disease and phenotype terms in its own database to produce a list of seed genes. These genes are then checked for interactions with all other genes in the genome, accounting for four types of interaction; protein-protein interactions, genes in the same Biosystem or the same gene family and in transcription interactions. Based on all of this, a machine learning model is used to generate a score for each gene, and the output is a list of genes ordered by their scores.  The output gene list for <i>lactic acidosis</i> appears in the left panel and the top 10 (this number is user configurable) genes appear as gene badges, ready to be interrogated.</p>

<div><img src="/public/images/blog/phenolyzer/phenolyzer-list.png" style="width:100%; margin:20px 0px;" /></div>

<p>By default, the genes are not automatically analyzed, but we can select 'Analyze all' to begin analysis. When complete, the gene badges describe any interesting variants within them using the glyphs described in the <a href="http://iobio.io/2016/03/28/multi-gene/" target="_blank">multi-gene analysis post</a>.</p>

<div><img src="/public/images/blog/phenolyzer/gene-badges.png" style="width:100%; margin:20px 0px;" /></div>

<p>The gene badges are in the prioritized order, so the gene to the furthest left (TTPA) is considered the most important by <a href="http://phenolyzer.usc.edu/" target="_blank">Phenolyzer</a>. The RAI1 gene still appears in the gene badges, since this was present prior to running Phenolyzer. We can see that there are no glyphs associated with the TTPA gene, so there are no high impact variants to look at there. The highest ranked gene that does have interesting variants to look at is PDHA1, so that is the first gene we should look at. In this gene, we see only two variants; an insertion in a promoter and a <i>de novo</i> missense variant predicted to be <i>deleterious</i> by SIFT and <i>probably_damaging</i> by PolyPhen. The variant is not seen in either 1000 Genomes or in ExAC, so this variant may well be a good candidate for the obseved phenotype.</p>

<div><img src="/public/images/blog/phenolyzer/pdha1.png" style="width:75%; margin:20px 100px;" /></div>

<p>We hope that you find this functionality useful and we would like to thank Kai Wang and his lab for allowing <a href="http://phenolyzer.usc.edu/" target="_blank">Phenolyzer</a> to be integrated into <a href="http://gene.iobio.io" target="_blank">gene.iobio</a>.</p>

  </div>

  
</div>

<div class="related posts">
  <h2>Related Posts</h2>
  <ul class="related-posts">
    
<!--       <li>
        <h3>
          <a href="/2017/09/29/gene_2.5.0/">
            ClinVar variants, gnomAD, affected status
            <small>29 Sep 2017</small>
            <div><img src=/public/images/blog/gene_2.5.0/banner.png/></div>
          </a>
        </h3>
      </li> -->
      <li class="snippet">
        <div class='snippet-thumbnail'><a href="/2017/09/29/gene_2.5.0/"><img src=/public/images/blog/gene_2.5.0/banner.png></img></a></div>
        <div class='snippet-text'>
          <div class='snippet-byline'>BY Tony DiSera ON Sep 29, 2017</div>
          <div></div>
          <img class='snippet-headshot' src="../../../../public/images/team/Tony_DiSera.png"/>
          <div class='snippet-title'><a href="/2017/09/29/gene_2.5.0/">ClinVar variants, gnomAD, affected status</a></div>
          <div class='snippet-subtitle'>What's new in gene.iobio 2.5.0</div>
        </div>
      </li>
    
<!--       <li>
        <h3>
          <a href="/2017/09/27/loading-data/">
            Loading data into gene.iobio
            <small>27 Sep 2017</small>
            <div><img src=/public/images/blog/loading_data/blog_image_data.jpg/></div>
          </a>
        </h3>
      </li> -->
      <li class="snippet">
        <div class='snippet-thumbnail'><a href="/2017/09/27/loading-data/"><img src=/public/images/blog/loading_data/blog_image_data.jpg></img></a></div>
        <div class='snippet-text'>
          <div class='snippet-byline'>BY Tony DiSera ON Sep 27, 2017</div>
          <div></div>
          <img class='snippet-headshot' src="../../../../public/images/team/Tony_DiSera.png"/>
          <div class='snippet-title'><a href="/2017/09/27/loading-data/">Loading data into gene.iobio</a></div>
          <div class='snippet-subtitle'>Loading your variant and alignment files</div>
        </div>
      </li>
    
<!--       <li>
        <h3>
          <a href="/2017/07/11/coverage/">
            Coverage analysis
            <small>11 Jul 2017</small>
            <div><img src=/public/images/blog/coverage/main.png/></div>
          </a>
        </h3>
      </li> -->
      <li class="snippet">
        <div class='snippet-thumbnail'><a href="/2017/07/11/coverage/"><img src=/public/images/blog/coverage/main.png></img></a></div>
        <div class='snippet-text'>
          <div class='snippet-byline'>BY Alistair Ward ON Jul 11, 2017</div>
          <div></div>
          <img class='snippet-headshot' src="../../../../public/images/team/Alistair_Ward.png"/>
          <div class='snippet-title'><a href="/2017/07/11/coverage/">Coverage analysis</a></div>
          <div class='snippet-subtitle'>Identifying problems with coverage</div>
        </div>
      </li>
    
  </ul>
</div>

    </div>

    <div class="footer">
  
    <div class="logos">
      <div style="position:absolute; left: 100px; top:0px; font-size:50px;font-family:'Overlock SC', cursive;"><a href="http://marthlab.org" style="color:black; text-decoration:none;">Marthlab</a></div>
      <a href="http://ucgd.genetics.utah.edu/"><img src="/public/images/ustar-ucgd-logo.jpg" style="height:60px;"/></a>
      <a href="http://www.genetics.utah.edu/"><img src="/public/images/genetics_mainlogo3_lrg.png" style="height:50px;position:absolute;right:0px;top:7px"/></a>
    </div>
  
  <div class="developers">
        <div class='title'>Developed in the <a href="http://marthlab.org/">Marth Lab by</a></div>

        <div class="members">
          <span style="margin-right:15px">
            <a href="https://github.com/chmille4"><img src="/public/images/github_2d8fc1.png"/></a>
            Chase Miller
          </span>
          <span style="margin-right: 15px">
            <a href="https://github.com/yiq"><img src="/public/images/github_2d8fc1.png"/></a>
            Yi Qiao
          </span>
          <span style="margin-right: 15px">
            <a href="https://github.com/tonydisera"><img src="/public/images/github_2d8fc1.png"/></a>
            Tony DiSera
          </span>
          <span style="margin-right: 15px">
            <a href="https://github.com/alistairnward"><img src="/public/images/github_2d8fc1.png"/></a>
            Alistair Ward
          </span>
        </div>
    </div>
</div>
  </body>
</html>
